### PR TITLE
Add spiechu symfony commons bundle

### PR DESCRIPTION
**Pull request by spiechu**
| Q             | A
| ------------- | ---
| License       | MIT

Bundle allows to validate JSON / XML responses against given JSON Schema / XSD files using simple controller annotations.
Also allows API versioning and provides compatibility with JMSSerializerBundle `since-version` & `until-version` attributes.

Notice that JMSSerializerBundle and FOSRestBundle are not yet fully compatible with Symfony 4.

### CODE REVIEW

File: spiechu/symfony-commons-bundle/0.1/config/packages/spiechu_symfony_commons.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+spiechu_symfony_commons:
+    get_method_override:
+        enabled:              false
+        #listener_service_id:  spiechu_symfony_commons.event_listener.get_method_override_listener
```
Comment (Nyholm) [inline spiechu/symfony-commons-bundle/0.1/config/packages/spiechu_symfony_commons.yaml]: Do we really need these comments?
Will they help the user or would it be better to refer them to the readme?

Comment (spiechu) [reply spiechu/symfony-commons-bundle/0.1/config/packages/spiechu_symfony_commons.yaml]: To be honest I was suggested by other already accepted recipes.
I saw configuration dumps fully commented out. I'm not sure what is better for DX.

Of course I can remove any commented out configuration and point user to readme.

Comment (Nyholm) [reply spiechu/symfony-commons-bundle/0.1/config/packages/spiechu_symfony_commons.yaml]: Hm, I see. 

The rule of thumb is: Do you think most (80%) of the users will change these values? If not, we should refer them to your docs instead.

**Comment by spiechu** [comment]
I've removed commented out configuration. I think PR is ready for final CR.

**Pull request merged**